apple/RNSVGUIKit.h

```diff
@@ -0,0 +1,41 @@
+// Most (if not all) of this file could probably go away once react-native-macos's version of RCTUIKit.h makes its way upstream.
+// https://github.com/microsoft/react-native-macos/issues/242
+
+#if !TARGET_OS_OSX
+
+#import <UIKit/UIKit.h>
+
+#define RNSVGColor UIColor
+#define RNSVGPlatformView UIView
+#define RNSVGTextView UILabel
+#define RNSVGView UIView
```

Comment:
What's the distinction between RNSVGView and RNSVGPlatformView?

Reply:
RNSVGView is meant to be an extension of RCTUIView and includes all the bells and whistles that we would expect from a React Native view. RNSVGPlatformView is an alias for a platform-specific view, which is useful for functions that are already built into AppKit/UIKit, such as traveling through the view hierarchy. (For example, look at `-[RNSVGView tintColor]` in RNSVGUIKit.macos.m.)

Reply:
But since RCTUIView inherits from UIView and NSView can we just use that in place of RNSVGPlatformView?

Reply:
RCTUIView is defined in RCTUIKit.h, which isn't available on plain react-native yet, so we can't use RCTUIView.